tests/clippy.toml

```diff
@@ -0,0 +1 @@
+# default config for tests
```

Comment:
Please extend this comment to explain that this file is necessary to override the root `clippy.toml` file.